### ISSUE TITLE
[k8s blog] series: SIG profiles/storytelling

### ISSUE THREAD

**Issue by parispittman** [kubernetes/community#3654]
put a story/human touch to our special interest groups. do interviews and tell stories around:

- their formation
- key challenges
- successes
- what they look for in new / retaining contributors
- funny / cool / TIL antidotes 
- shoutout individuals / profile the folks doing the work

List of community groups for reporting:
- [x] scalability (@vonguard ref: [19628](https://github.com/kubernetes/website/pull/19628))
- [x] wg-k8s-infra (@celanthe; finished)
- [ ] network (needs someone to pick up) 
- [ ] node (needs someone to pick up)
- [ ] cluster life (needs someone to pick up)
- [ ] [API Machinery](sig-api-machinery/README.md)
- [ ] [Apps](sig-apps/README.md)
- [ ] [Architecture](sig-architecture/README.md)
- [ ] [Auth](sig-auth/README.md)
- [ ] [Autoscaling](sig-autoscaling/README.md)
- [ ] [CLI](sig-cli/README.md)
- [ ] [Cloud Provider](sig-cloud-provider/README.md)
- [ ] [Contributor Experience](sig-contributor-experience/README.md)
- [ ] [Docs](sig-docs/README.md)
- [ ] [Instrumentation](sig-instrumentation/README.md)
- [ ] [Multicluster](sig-multicluster/README.md)
- [ ] [Release](sig-release/README.md)
- [ ] [Scheduling](sig-scheduling/README.md)
- [ ] [Service Catalog](sig-service-catalog/README.md)
- [ ] [Storage](sig-storage/README.md)
- [ ] [Testing](sig-testing/README.md)
- [ ] [UI](sig-ui/README.md)
- [x] [Usability](sig-usability/README.md)
- [x] [Windows](sig-windows/README.md)

**Comment by onlydole** [comment]
Template for questions [has been started](https://docs.google.com/document/d/1mqC5ySqtAENvhDXGnMHDFWjlP7AXJdyIE0PngeHhb0I/edit?usp=sharing) - let me know if you need edit access, or want to see anything else from these blog posts.

**Comment by celanthe** [comment]
I reached out to the k8s-infra working group today to see about getting an interview on the books sometime over the next two weeks, with a goal of publishing our next blog post sometime over the next 2-3 weeks. :)

**Comment by samudralavamshi54** [comment]
Reached out to Sig Cluster life cycle posted it on their slack for interview time if not shared the template doc, will follow up and update.

Thanks to @onlydole @celanthe for templating the required communication

**Comment by samudralavamshi54** [comment]
Got the responses from SIG cluster lifecycle, will  work with @onlydole to format it .

**Comment by samudralavamshi54** [comment]
reached out to next group SIG node will update

**Comment by onlydole** [comment]
Thank you, @samudralavamshi54! Feel free to raise up in Slack and I'll be happy to help!

**Comment by celanthe** [comment]
An update on this for those that can't attend today's meeting 4/3/20, got all feedback from SIG chairs, and am going to draft this blog post with the goal of publishing it in mid-April sometime. :)

**Comment by mbbroberg** [comment]
SIG Windows is claimed! @kaslin has that as a new contributor to upstream marketing team.

Tips to get started: 

- Jump in the Slack channel and offer to get on the phone and talk through what's going on
- "Is there something that your SIG is really looking for right now?" 
  - Contributors? Users? Awareness? 
  - Whatever they offer ends up as the call to action for the article
- Start writing! Wherever works for you: Markdown preferred if you're familiar (Hackmd.io or a PR), though Google Docs are always welcome as well

**Comment by celanthe** [comment]
Another update from me--The first post in my K8s-infra-wg blog series has been sent for review to @onlydole and @mbbroberg :)

**Comment by mbbroberg** [comment]
Here's the latest update on available writing. cc @celanthe @kaslin @vonguard. 

- [x] scalability ([@vonguard](https://github.com/vonguard); finished)
- [x] wg-k8s-infra ([@celanthe](https://github.com/celanthe); finished)
- [x] [Windows](https://github.com/kubernetes/community/issues/sig-windows/README.md) by @kaslin
- [ ] [Network](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle) 
- [ ] [Node](https://github.com/kubernetes/community/tree/master/sig-node)
- [ ] [Cluster lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle)
- [ ] [API Machinery](https://github.com/kubernetes/community/issues/sig-api-machinery/README.md)
- [ ] [Apps](https://github.com/kubernetes/community/issues/sig-apps/README.md)
- [ ] [Architecture](https://github.com/kubernetes/community/issues/sig-architecture/README.md)
- [ ] [Auth](https://github.com/kubernetes/community/issues/sig-auth/README.md)
- [ ] [Autoscaling](https://github.com/kubernetes/community/issues/sig-autoscaling/README.md)
- [ ] [CLI](https://github.com/kubernetes/community/issues/sig-cli/README.md)
- [ ] [Cloud Provider](https://github.com/kubernetes/community/issues/sig-cloud-provider/README.md)
- [ ] [Contributor Experience](https://github.com/kubernetes/community/issues/sig-contributor-experience/README.md)
- [ ] [Docs](https://github.com/kubernetes/community/issues/sig-docs/README.md)
- [ ] [Instrumentation](https://github.com/kubernetes/community/issues/sig-instrumentation/README.md)
- [ ] [Multicluster](https://github.com/kubernetes/community/issues/sig-multicluster/README.md)
- [ ] [Release](https://github.com/kubernetes/community/issues/sig-release/README.md)
- [ ] [Scheduling](https://github.com/kubernetes/community/issues/sig-scheduling/README.md)
- [ ] [Service Catalog](https://github.com/kubernetes/community/issues/sig-service-catalog/README.md)
- [ ] [Storage](https://github.com/kubernetes/community/issues/sig-storage/README.md)
- [ ] [Testing](https://github.com/kubernetes/community/issues/sig-testing/README.md)
- [ ] [UI](https://github.com/kubernetes/community/issues/sig-ui/README.md)
- [ ] [Usability](https://github.com/kubernetes/community/issues/sig-usability/README.md)

**Comment by celanthe** [comment]
@mbbroberg I would be very interested in taking on a piece about SIG-Docs after I finish my next post in the series on the Infra WG. I can tentatively commit to this in August, I am aiming for a one-post-per-month cadence currently as my bandwidth allows.

**Comment by nyuday** [comment]
@mbbroberg I’d like to try creating a piece about SIG-Usability.

**Comment by vonguard** [comment]
@mbbroberg I'm still interested in writing about SIG Networking but I was completely unable to get anyone on the team to schedule an interview

**Comment by mbbroberg** [comment]
Hey all! The team at contrib comms is working to shift this issue toward a system that fits into our editorial workflow. The objective is to make a document that helps facilitate asynchronous interviews with SIGs so profiling takes a little less effort for new authors. 

Here is where we're putting that together: https://docs.google.com/document/d/17zSSDRYoVySbxev2_9r1x89YzFtRH-5DUrz-Q3VAGuo/edit

It will include the thoughtful questions Taylor put together and link to the great articles already published on this topic. The goal will be to add a non-issue based tracker only once someone has a draft in progress to this project: https://github.com/orgs/kubernetes/projects/41

When we get it done, my thought is we can close this issue and keep this program (of SIG profiling) as a standard practice of our Upstream Marketing group. 

Feedback or edits welcome of course! Feel free to add questions, notes, or start to structure the doc if you get there before @kaslin and I get a chance to do so.

**Comment by mbbroberg** [comment]
We have a new issue to complete the shift to an async format. Thank you to everyone who's been helped the community through these articles to date! 

/close
